.github/dependabot.yml

```diff
@@ -7,42 +7,42 @@ updates:
     timezone: UCT
```

Comment:
leaving docker updates as daily to get notified of point release with sec. fixes